### ISSUE TITLE
Add --version switch to all tools

### ISSUE THREAD

**Issue by tseemann** [phac-nml/mob-suite#35]
```
% mob_typer --version 2> /dev/null
mob_typer 2.0.2

% echo $?
0
```
print to stdout  and return error code 0

**Comment by EricDeveaud** [comment]
+1 
as you have a mob_suite.version you may want to add something like that 

parser.add_argument('--version', action='version',
                    version='%(prog)s {version}'.format(version=__version__))

**Comment by tseemann** [comment]
@EricDeveaud - that is the simplest way to do it, thank you!

**Comment by EricDeveaud** [comment]
> I don't see a PR or Commit for this yet?

see: https://github.com/phac-nml/mob-suite/pull/37

**Comment by kbessonov1984** [comment]
We've added the -V switch in the latest PR and accommodated the taxa.sqlite request to be in databases folder. We are tackling several issues in the PR. I am doing final tests and will release version 2.0.2 in a couple of hours. Thank you for your help.

**Comment by kbessonov1984** [comment]
James added ` -V, --version         show program's version number and exit` switch in all the tools as requested. Try it out in `2.0.2` release. E.g.

```
$mob_typer --version
mob_typer (2.0.2)
```

**Comment by tseemann** [comment]
Please remove the brackets. That is not GNU standard, and will cause parsing issues in pipeline audits. The example I gave at the top is what it should be.

**Comment by jrober84** [comment]
This issue has been resolved in release 2.1.0, thanks for your help @tseemann

**Comment by tseemann** [comment]
Thank you @jrober84
